acra/configuring-maintaining/metrics/_index.md

```diff
@@ -1,4 +1,75 @@
 ---
+
+weight: 1
 title: Metrics
 bookCollapseSection: true
 ---
+
+# Metrics in Acra
+
+Starting with version [`0.83.0`](https://github.com/cossacklabs/acra/releases/tag/0.83.0), Acra supports tracking and exporting of the basic metrics of AcraServer, AcraConnector, and AcraTranslator to [` Prometheus`](https://prometheus.io/).
+
+
+## Metrics configuration
+
+### Command line flags
+
+* `--incoming_connection_prometheus_metrics_string=<url>`
+
+  URL (tcp://host:port) which will be used to expose Prometheus metrics (`<URL>/metrics` address to pull metrics).
+
+
+### Exposed metrics
+
+Acra can expose standard set of metric [`types`](https://prometheus.io/docs/concepts/metric_types/) for AcraServer, AcraConnector, and AcraTranslator. 
+Additional metrics registration info could be found in [`prometheus.go`](https://github.com/cossacklabs/acra-Q12021/blob/master/cmd/acra-server/common/prometheus.go) files located in `cmd` folder for each corresponded Acra component.
```

Comment:
can we refer to cossacklabs/acra or we added new metrics in acra-ee?

Reply:
I think no. Will reference to `cossacklabs/acra `